ui/src/app/cohort-search/demographics/demographics.component.ts

```diff
@@ -143,10 +145,13 @@ export class DemographicsComponent implements OnInit, OnChanges, OnDestroy {
             .subscribe(sel => this.noSelection = sel)
         );
 
-        this.subscription.add(this.ngRedux
-            .select(totalCount)
-            .subscribe(participantCount => this.count = participantCount)
-        );
+
+      this.subscription = this.count$
+        .subscribe(item => {
```

Comment:
nit: 'item' doesn't make sense here, maybe name it 'count' or something similar to avoid confusion